src/sentry/interfaces/base.py

```diff
@@ -50,6 +73,7 @@ class Interface(object):
     ephemeral = False
 
     def __init__(self, **data):
+        self.is_processed_data = data.pop('is_processed_data', True)
```

Comment:
fwiw if you define this on the class as a default (e.g. is_processed_data = True) I think you can avoid the __getattr__ hack